### PR TITLE
[ko] Synchronize install-kubectl-linux.md

### PR DESCRIPTION
**Pull request by bitfree**
correct some typos to work correctly for ko docs

### CODE REVIEW

File: content/ko/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -121,12 +121,15 @@ card:
...
    ```shell
    sudo curl -fsSLo /etc/apt/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+   sudo chmod 644 /etc/apt/keyrings/kubernetes-apt-keyring.gpg # 키링 값을 읽을 수 있는 권한을 부여한다
    ```
 
 3. 쿠버네티스 `apt` 리포지터리를 추가한다.
 
    ```shell
-   echo "deb [signed-by=/etc/apt/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+   # This overwrites any existing configuration in /etc/apt/sources.list.d/kubernetes.list
```
Comment (sftim) [inline content/ko/docs/tasks/tools/install-kubectl-linux.md:130]: I notice that this line is not in Korean.

File: content/ko/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -121,12 +121,15 @@ card:
...
+   echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/ /' | sudo tee /etc/apt/sources.list.d/kubernetes.list
+   sudo chmod 644 /etc/apt/sources.list.d/kubernetes.list   # helps tools such as command-not-found to work correctly
```
Comment (sftim) [inline content/ko/docs/tasks/tools/install-kubectl-linux.md:132]: We could localize the comment.

**Review by sftim — COMMENTED** [review]
/retitle [ko] Synchronize install-kubectl-linux.md

**Comment by seokho-son** [comment]
Ping @bitfree :)

**Pull request closed without merging**